File: server/src/main/java/ru/vitaliy/petrov/server/forms/requests/caraccident/CarAccidentUpdateRequest.java
package ru.vitaliy.petrov.server.forms.requests.caraccident;

import lombok.Data;

import javax.validation.constraints.NotEmpty;
import javax.validation.constraints.NotNull;
import java.sql.Date;
import java.sql.Time;

@Data
public class CarAccidentUpdateRequest {

    @NotNull
    @NotEmpty
    private final String updatedCarAccidentScene;

    @NotNull
    private final Date updatedCarAccidentDate;

    @NotNull
    private final Time updatedCarAccidentTime;

}

File: client/app/src/main/java/com/petrov/vitaliy/caraccidentapp/domain/models/requests/accident/CarAccidentUpdateRequest.kt
package com.petrov.vitaliy.caraccidentapp.domain.models.requests.accident

import java.sql.Date
import java.sql.Time

data class CarAccidentUpdateRequest(
    val updatedCarAccidentScene: String,
    val updatedCarAccidentDate: Date,
    val updatedCarAccidentTime: Time
)
